core/src/main/scala/com/mesosphere/usi/core/SpecState.scala

```diff
@@ -0,0 +1,8 @@
+package com.mesosphere.usi.core
+import com.mesosphere.usi.core.models.{PodId, PodSpec}
+
+case class SpecState(podSpecs: Map[PodId, PodSpec])
```

Comment:
nit: do we need a case class wrapper? can it be a
```suggestion
type SpecState = Map[PodId, PodSpec]
```

Reply:
We'll have other specs in here, reservation specs